prog/dftb+/lib_dftbplus/main.F90

```diff
@@ -941,7 +941,7 @@ subroutine processGeometry(env, iGeoStep, iLatGeoStep, tWriteRestart, tStopScc,
           end if
         end if
 
-        if (allocated(dispersion)) then
+        if (allocated(dispersion) .and. .not. tConverged) then
```

Comment:
Don't we need the same in the other branch of the reks-if?

Reply:
Just changed this as well (and probably fixed energy summation for REKS in this case during SCC cycles).